src/main/java/hlf/java/rest/client/controller/ChaincodeOperationsController.java

```diff
@@ -57,6 +58,9 @@ public ResponseEntity<String> getCurrentPackageId(
       @RequestParam("network_name") @Validated String networkName,
       @RequestParam("chaincode_name") @Validated String chaincodeName,
       @RequestParam("chaincode_version") @Validated String chaincodeVersion) {
+    networkName = StringEscapeUtils.escapeHtml4(networkName);
+    chaincodeName = StringEscapeUtils.escapeHtml4(chaincodeName);
+    chaincodeVersion = StringEscapeUtils.escapeHtml4(chaincodeVersion);
```

Comment:
Since xss protection is enabled in the header config, you can remove this sanitization. Kindly remove and re-run the scanner to confirm.

Reply:
I removed this, and re-ran the scan, but the scan threw the XSS errors. Then i included it and re-ran. The errors were not shown this time. Post this testing I raised the PR. So the scan requires this.